content/concepts/multiplex/early-negotiation.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Early Multiplexer Negotiation"
+description: "Peers can use security protocol extenstions for early muxer negotiation."
+weight: 162
+---
+
+## Typical connection upgrade process
+
+Peers upgrade raw transport connections by using the same
+[multistream-selection](https://github.com/multiformats/multistream-select)
+protocol to negotiate security and stream multiplexing.
```

Comment:
@p-shahi's suggestion sounds good to me. The protocol is called multistream-select though.

Reply:
We already explain the upgrade process in the overview (https://github.com/libp2p/docs/pull/270) as part of the muxer section. So I'd suggest we improve that rather than repeating the upgrade process here.

Reply:
My preference is to keep it here since it's a separate page.